### ISSUE TITLE
Kerbals on EVA spawn several meters away from the vessel

### ISSUE THREAD

**Issue by scimas** [mockingbirdnest/Principia#1452]
Since Cauchy, kerbals on EVA from a vessel in orbit spawn several meters away from the vessel. Doesn't happen when EVA-ing on ground or in atmosphere.

**Comment by pleroy** [comment]
Thanks for the report.  We were able to reproduce it.  Will investigate more.

**Comment by eggrobin** [comment]
We tested EVA when solving #1415, so the branch for #1428 still worked; however this was branched off master before #1426 was merged, and that one may affect new vessels.
I will check whether we skip some frames on the newly-added Kerbal.

**Comment by eggrobin** [comment]
Update: I do not see the message ```Skipping AdvanceTime and SetBodyFrames while waiting for the vessel to be fully ready (see #1421).``` in the log when reproducing the issue, so it is probably something else.

**Comment by eggrobin** [comment]
See the following link for the diff of the adapter between the state when we tested solved the ladder bug (where EVA still worked) and Cauchy: https://github.com/mockingbirdnest/Principia/compare/b7f6b8644b2a4c37b72ea26af5a8b6153c408bd9...2017062403-Cauchy.
Something that stands out there is the fix for #1307. I will try to determine whether that is the issue.

**Comment by eggrobin** [comment]
That seems to be it, I'll have to figure out why it breaks EVA spawning though, because we still need #1307 to stay fixed.

**Comment by eggrobin** [comment]
Ok, the explanation is that when an EVA Kerbal is created, it ignores collisions with the ship that spawned it for a little while. This appears to be done by nudging it back to the airlock in the same way we nudge things, after yielding WaitForFixedUpdate; since we expect nothing to run there, random things happen (it happens that nudging the transforms of the rigid bodies nudges the airlocks, while nudging the rigid bodies themselves did not, which is why the fix for #1307 uncovered this).
Since KSP is taking care of moving that Kerbal around, we don't need to move it, so this is solved by adding an unmanageability reason:
```c#
    if (vessel.isEVA && !vessel.evaController.Ready) {
      reasons.Add("vessel is an unready Kerbal");
    }
```
The Kerbal becomes known to Principia and is nudged exclusively by us once it is ready.

I will make a CL tomorrow to fix this.
